### PR TITLE
sched: Refine the comment for SCHED_INSTRUMENTATION_EXTERNAL

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
## Summary
follow up PR: https://github.com/apache/incubator-nuttx/pull/1746

### CODE REVIEW

**Review by davids5 — APPROVED** [review]
Works for me

**Comment by xiaoxiang781216** [comment]
@patacongo do you have any concern?

File: sched/Kconfig [hunk 1]
```diff
@@ -920,7 +920,13 @@ config SCHED_INSTRUMENTATION_EXTERNAL
 	bool "System performance monitor endpoints are external"
 	default n
 	---help---
-		Monitor only CPUs in the bitset.  Bit 0=CPU0, Bit1=CPU1, etc.
+		When this option isn't selected, the common code will implement
```
Comment (davids5) [inline sched/Kconfig:923]: This should say what will happen when it is enabled.

**Review by davids5 — CHANGES_REQUESTED** [review]
@xiaoxiang781216 - I think these changes are fine. I think you need to make the point that the API it either fully discriminated or a single end point that can A) act an in interface to a transport of B) will needs to de-multiplex the data to recreate the endpoints if SCHED_INSTRUMENTATION_EXTERNAL is not set.

File: sched/Kconfig (hunk 1 above)
Comment (xiaoxiang781216) [reply sched/Kconfig]: Done.

**Review by davids5 — APPROVED** [review]

**Comment by davids5** [comment]
@xiaoxiang781216 - since you asked for @patacongo 's input, we should give him time to respond. If we do not hear back by say mid Monday his time, we can assume he is OK with this level of the change. Then ping me and I will merge it.

**Comment by davids5** [comment]
I am assuming there was not objection on the naming front.

**Pull request merged**